abr_control/arms/jaco2/config.py

```diff
@@ -60,6 +60,8 @@ def __init__(self, hand_attached=False, **kwargs):
                        * np.sqrt(self.N_JOINTS))
                 }
 
+        self.START_POS = np.array(
```

Comment:
useful to have a safe starting position standardized for each arm

Reply:
hmmm i think this should vary by task and not necessarily always be the same for a given arm

Reply:
I think of it as a position that can be used as a default if you don't have one